mayastor-test/e2e/node_disconnect/node_disconnect_iscsi/node_disconnect_iscsi_test.go

```diff
@@ -0,0 +1,128 @@
+package node_disconnect_iscsi_test
+
+import (
+	"e2e-basic/common"
+	"fmt"
+	"os/exec"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var defTimeoutSecs = "90s"
+
+var g_nodeToKill = ""
+var g_nexusNode = ""
+var g_uuid = ""
+
+func disconnectNode(vmname string, nexusNode string) {
+	cmd := exec.Command("bash", "../../common/io_connect_node.sh", vmname, nexusNode, "I")
+	cmd.Dir = "./"
+	_, err := cmd.CombinedOutput()
+	if false {
+		Expect(err).ToNot(HaveOccurred())
+	}
+}
+
+func reconnectNode(vmname string, nexusNode string) {
+	cmd := exec.Command("bash", "../../common/io_connect_node.sh", vmname, nexusNode, "D")
+	cmd.Dir = "./"
+	_, err := cmd.CombinedOutput()
+	if false {
+		Expect(err).ToNot(HaveOccurred())
+	}
```

Comment:
This will never execute

Reply:
This and above, that was a temporary change I meant to take out. I'll fix.